tests/unittests/import_export_tests.py

```
@@ -1758,16 +1817,19 @@ def test_add_yolo_labels(self):
     @drop_datasets
     def test_add_coco_labels(self):
         dataset = self._make_dataset()
+
         classes = dataset.distinct("predictions.detections.label")
+        categories = [{"id": i, "name": l} for i, l in enumerate(classes, 1)]
```

Ruff

Ambiguous variable name: `l`

(E741)